dimod/binary/binary_quadratic_model.py

```diff
@@ -2423,8 +2423,17 @@ def Binary(label: Optional[Variable] = None, bias: Bias = 1,
         dtype: Data type for the returned binary quadratic model.
 
     Returns:
-        Instance of :class:`.BinaryQuadraticModel`.
+        A :class:`.BinaryQuadraticModel` representing a binary variable.
 
+    Examples:
+        This example generates a BQM to represent the polynomial,
+        :math:`x^2 + 3x -1.5`, where :math:`x` is a binary variable with values
+        :math:`\{0, 1\}`. Remember that for such variables, :math:`x^2 = x`.
```

Comment:
Much better. Though I am surprised that you wanted to introduce the `x^2 == x` thing here. I think just `3*x - 1.5` would be more than sufficient.

Reply:
I deliberated on that but saw a couple of advantages: (1) it makes the example's output more interesting (2) it shows/reminds users of an extremely important property of binary variables that someone who isn't immersed might miss. I figured it might save a few Support questions.
I can remove if you think it distracts in a harmful way.

Reply:
I've thought about it more and I do think it distracts. When I see examples I usually just skip the the code rather than reading the text. I think in this case we want the code to be extremely self-explanatory. The self-loops thing is an excellent topic for the "gotchas" section of the more full symbolic docs